### DISCUSSION TITLE
Marlin Firmware -  Octoprint M112 Shutdown

### DISCUSSION THREAD

**Discussion by Dlebla8** [Jyers/Marlin#777] [Q&A] [answered]
First, LOVE this firmware, if I can figure this issue out will never have to leave it.

Environment:  
Ender 3 V2 4.22 main board 
Octoprint Version 1.6.1
BLTouch 
Marlin 2.0.1 

Issue:  As soon as Octoprint tries to connect printer throws an M112

Any suggestions on what to check, how to fix  I want this Firmware.

Dale
(The Master of Dragons and keeper of Eagles)

**Comment by smooreace** [comment]
A little search for the error code will answer all your questions

**Comment by Jyers** [comment, marked as answer]
https://github.com/Jyers/Marlin/wiki/OctoPrint-Settings

**Reply by Dlebla8** [reply]
Jyers, 

          Actually, A wipe of my Cura and a new install of Octoprint cleared up the issue and some others.  Not sure yet what happen to the Vanilla installs, but obviously something did.

          The Firmware is awesome, I am working on a Menu map ( so I know where to find everything) and will pass it along when I am done.
